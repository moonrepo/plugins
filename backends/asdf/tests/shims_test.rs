#[cfg(not(windows))]
mod asdf_backend {
    use proto_pdk_test_utils::*;

    generate_shims_test!("asdf:zig", ["zig"]);
}
