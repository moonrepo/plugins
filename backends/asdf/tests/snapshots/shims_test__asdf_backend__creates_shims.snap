---
source: backends/asdf/tests/shims_test.rs
expression: "std :: fs ::\nread_to_string(sandbox.path().join(\".proto/shims/registry.json\")).unwrap()"
---
{
  "zig": {}
}
